src/leappto/cli.py

```diff
@@ -780,8 +785,26 @@ def print_migrate_info(text):
             None
         )
 
-        return_code, claimed_names = mc.check_target()
-        for name in sorted(claimed_names):
+        check_result = {}
+        return_code, check_result['containers'] = mc.check_target_containers()
+
+        if parsed.status:
+            check_commands = dict()
```

Comment:
I'd prefer to keep all the logic here inside the MigrationContext, so I'd suggest moving the current `check_target` to `check_target_containers` (as you've already done), but then re-using the `check_target` method name for the full status check.

The CLI logic would then look like:

```
if parsed.status:
    return_code, target_status = mc.check_target()
    print(dumps(target_status))
    sys.exit(return_code)

return_code, claimed_names = mc.check_target_containers()
# Existing code
```